deploy/azure/app/function/constraints.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.1"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.98"
```

Comment:
is this the latest version we can use?

Reply:
Most likely not but I've keep it consistent with the rest of the Terraform in the repo. There are also a bunch of warnings produced when we run it. I'll create another story to upgrade providers and address warnings